ConwaysGameOfLife/Universe.cs

```diff
@@ -67,23 +65,25 @@ public List<Location> GetCellNeighbourLocations(Cell cell)
             return neighbourLocations;
         }
 
-        public CellState GetCellStateFromLocation(Location location)
+        public CellState GetCellStateFromLocation(Location location) //TODO: do I need this - can use get cellaatlocation
         {
-            var newCells = Cells.Where(cell => cell.Location == new Location(cell.Location.X, cell.Location.Y));
-            var newCell = newCells.ElementAt(0);
-            return newCell.CellState;
+            // var newCell = Cells.Find(cell => cell.Location == new Location(cell.Location.X, cell.Location.Y));
+            var x = location.X;
+            var y = location.Y;
+            var cellAtLocation = Cells.FirstOrDefault(cell => cell.Location.X == x && cell.Location.Y == y);
+            return cellAtLocation.CellState;
         }
 
         public List<Cell> CheckIfNeighboursAlive(Cell cell) 

```

Comment:
The name implies that we want to know if the neighbours are alive
- do we need to return a list of cells? is that what we're expecting? should it return a boolean instead?
- is it all neighbours are alive?
- is it some neighbours are alive?
- is it one neighbour is alive?

What do we need this function for? Probably because later on, we want to implement rules that are based on the number of alive neighbours.
```suggestion
        public int CountAliveNeighbours(Cell cell)
```